app/lib/features/invite_members/pages/invite_page.dart

```diff
@@ -171,14 +201,131 @@ class InvitePage extends ConsumerWidget {
               style: Theme.of(context).textTheme.bodySmall,
             ),
             const SizedBox(height: 20),
-            ActerPrimaryActionButton(
-              onPressed: () =>
-                  EasyLoading.showInfo(L10n.of(context).comingSoon),
-              child: Text(L10n.of(context).generateInviteCode),
-            ),
+            _inviteCodeUI(),
           ],
         ),
       ),
     );
   }
+
+  Widget _inviteCodeUI() {
```

Comment:
I think there is enough within this widget and all the following functions allow to separate that out into its own widget file...